index.bs

```diff
@@ -1237,7 +1237,7 @@ To <dfn export for="SendStream" lt="create|creating">create</dfn> a
    [=WritableStream/set up/abortAlgorithm=] set to |abortAlgorithm|.
 1. [=AbortSignal/Add=] the following steps to |stream|'s \[[controller]]'s \[[signal]].
   1. If |stream|'s [=[[PendingOperation]]=] is null, then abort these steps.
-  1. Let |reason| be |stream|'s \[[controller]]'s \[[abortReason]].
+  1. Let |reason| be |stream|'s \[[controller]]'s \[[signal]]'s \[[reason]].
```

Comment:
How about \[[signal]]'s [=AbortSignal/reason=]?

Reply:
That was also my initial idea, but then thought maybe I should keep it consistent with just referring to the internal slot. However, now thinking again, using [=AbortSignal/abort reason=] makes more sense to me, so I've changed it :)